dali/operators/expressions/expression_impl_factory_gpu.cu

```diff
@@ -24,8 +24,9 @@
 #define ALLOWED_TYPES \
   (uint8_t, uint16_t, uint32_t, uint64_t, int8_t, int16_t, int32_t, int64_t, float, double)
 
-#define ALLOWED_OPS \
-  (ArithmeticOp::add, ArithmeticOp::sub, ArithmeticOp::mul, ArithmeticOp::div, ArithmeticOp::mod)
+#define ALLOWED_OPS                                                                                \
```

Comment:
how about defining `ALLOWED_TYPES` and `ALLOWED_OPS` only once?

Reply:
It's done in next PR, where I merged the factories.